tests/Aspire.Hosting.Redis.Tests/RedisFunctionalTests.cs

```diff
@@ -293,4 +366,14 @@ public async Task PersistenceIsDisabledByDefault()
             await app.StopAsync();
         }
     }
+
+    internal sealed class RedisInsightDatabaseModel
```

Comment:
What is this based on?

Reply:
this is the database model that Redis Insight API returns.
I used this to check the Redis connection.

```csharp
var testDatabaseConnectionApiUr = $"{insightEndpoint.Scheme}://{insightEndpoint.Host}:{insightEndpoint.Port}/api/databases/test/{db.Id}";
var cts = new CancellationTokenSource(TimeSpan.FromSeconds(2));
var testConnectionResponse = await client.GetAsync(getDatabasesApiUrl, cts.Token)
    .ConfigureAwait(false);
response.EnsureSuccessStatusCode();
```

The `db.Id` is one of the properties of this class.